app/controllers/posts_controller.rb

```diff
@@ -1,9 +1,26 @@
 class PostsController < ApplicationController
   def index
-    puts params
+    param_id = params['id'].to_i
+    @single_user_found = User.find_by(id: param_id)
+    @user_found_posts = Post.where(author_id: param_id)
+    @all_comments = Comment.all
   end
 
   def show
     puts params
+    param_id = params['id'].to_i
+    param_user_id = params['user_id'].to_i
+    @user_found = User.find_by(id: param_user_id)
+    @post = Post.find_by(id: param_id)
+    @all_comments = Comment.where(post_id: param_id)
+  end
+
+  def new
+    Post.new
+  end
+
+  def create
+    Post.create(title: params['title'], text: params['message'], author_id: params['author_id'])
```

Comment:
- [ ] It's a requirement of this project to make use of strong parameters as requested by the previous reviewer. These parameters should be made private

```.ruby
private 
    def post_params
      params.require(:post).permit(:title, :text)
    end
```